src/Transports/MassTransit.RabbitMqTransport/Configuration/IRabbitMqStreamConfigurator.cs

```diff
@@ -21,6 +21,11 @@ public interface IRabbitMqStreamConfigurator
     /// </summary>
     long MaxSegmentSize { set; }
 
+    /// <summary>
+    /// Set the stream filter value for the consumer
+    /// </summary>
+    void UsingFilter(string filter);
```

Comment:
Any reason this isn't just a `string Filter { set; }` property? If there aren't any other options related to the filter configuration, seems cleaner.

Reply:
Either way should be fine, I decided to stay consistent with the other consumer arguments in that class, which all use methods to set their values.

Reply:
Well, the rest are properties. Perhaps I should have been more clear, I would like it changes to a property.